tests/test_validator.py

```diff
@@ -35,6 +35,18 @@ async def test_verify_valid_utterances():
     assert validator.verify_utterances()
 
 
+async def test_verify_valid_responses():
+    importer = RasaFileImporter(
+        domain_path="data/test_domains/selectors.yml",
+        training_data_paths=[
+            "data/test_selectors/nlu.yml",
+            "data/test_selectors/stories.yml",
```

Comment:
Based on the previous comment, we might remove the `data/test_selectors/stories.yml` from here.